guava/src/com/google/common/hash/BloomFilter.java

```diff
@@ -33,6 +34,7 @@
 import java.io.OutputStream;
 import java.io.Serializable;
 import java.math.RoundingMode;
+import java.util.concurrent.atomic.AtomicLongArray;
```

Comment:
I don't think you are using this down, considering the `LockFreeBitArray.data.set` method just takes long input to set the data.

Reply:
Ah yeah, will remove it.